scaling_performance/optimizing_storage.adoc

```diff
@@ -145,7 +145,9 @@ Corruption may occur when using NFS to back {production-title} scaled/HA registr
 In an {product-title} hosted metrics cluster deployment:
 
 * The preferred storage technology is block storage.
-* It is not recommended to use NAS storage (excluding CNS/CRS GlusterFS as it uses a block storage interface from iSCSI) for hosted metrics cluster deployment with production workloads.
+* It is not recommended to use NAS storage (excluding CNS/CRS GlusterFS as it uses
+a block storage interface from iSCSI) for hosted metrics cluster deployment with
+production workloads.
```

Comment:
Something seems missing here:
for **a** hosted metrics cluster deployment
for hosted metrics cluster deployment**s**